src/components/Breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,148 @@
+<!--
+ - @copyright Copyright (c) 2020 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @author Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<docs>
+
+### General description
+
+This component is meant to be used inside a Breadcrumbs component.
+
+</docs>
+
+<template>
+	<div class="crumb"
+		ref="crumb"
+		:class="{over: hovering}"
+		draggable="false"
+		@dragstart="dragstart"
+		@drop="dropped($event)"
+		@dragover="dragOver($event)"
+		@dragenter="($event) => dragEnter($event)"
+		@dragleave="dragLeave">
+		<element :is="tag" :to="to" :href="href">
+			<span v-if="icon" :class="icon" class="icon" />
+			<span v-else>{{ title }}</span>
+		</element>
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'Breadcrumb',
+	props: {
+		/**
+		 * The displayed title of the breadcrumb.
+		 */
+		title: {
+			type: String,
+			required: true,
+		},
+		/**
+		 * The router-link to prop [https://router.vuejs.org/api/#to](https://router.vuejs.org/api/#to)
+		 * If set, the breadcrumbs will be rendered by router-link.
+		 */
+		to: {
+			type: String,
+			required: false,
+			default: undefined,
+		},
+		/**
+		 * Set this prop if your app doesn't use vue-router, breadcrumbs will show as normal links.
+		 */
+		href: {
+			type: String,
+			required: false,
+			default: undefined,
+		},
+		/**
+		 * Set a css icon-class to show an icon instead of the title text.
+		 */
+		icon: {
+			type: String,
+			required: false,
+			default: '',
+		},
+	},
+	data: function() {
+		return {
+			hovering: false,
+		}
+	},
+	computed: {
+		/**
+		 * determines which element tag to use
+		 * @returns {String} the tag
+		 */
+		tag() {
+			return this.to ? 'router-link' : 'a'
+		},
+	},
+	methods: {
+		cancelEvent(e) {
+			e.stopPropagation()
+			e.preventDefault()
+			return false
+		},
+		dragstart(e) {
+			return this.cancelEvent(e)
```

Comment:
`e.preventDefault()` is necessary so it doesn't navigate to the crumb when you drop and to prevent dragging crumbs.

`e.stopPropagation()` doesnt' seem to be necessary. I removed it.

Reply:
Ah, I thought I posted it too, but you can avoid using prevent/stop with the events modifier directly
https://vuejs.org/v2/guide/events.html#Event-Modifiers

Reply:
I added the `.prevent` modifier to the `Breadcrumb` component template. For the `Breadcrumbs` render function modifiers are not available and one should handle it in the handler instead https://vuejs.org/v2/guide/render-function.html#The-Data-Object-In-Depth